CONTRIBUTING.md

```diff
@@ -108,6 +108,27 @@ resulting ``_build`` directory it produces; ``index.html`` is the main page.
 **NOTE:** You can try to use the `make` commands on Windows if you have `make` installed and something like `Cygwin`, `MSYS2` or `MYSYS`, or by using Windows Subsystem for Linux (`WSL`).
 
 
+
+## Configuring Netlify
+
+The steps to configure Netlify to preview pull requests are the following:
+
+* Pass a command to build the docs. In our case that command is
+
+      ci/install.sh && make docs
+
+* Pass the root directory of the generated docs. In our case that is
+
+      doc/_build/html/
```

Comment:
~~~suggestion
1. Pass a command to build the docs:

```bash
ci/install.sh && make docs
```

2. Pass the root directory of the generated docs:

```bash
doc/_build/html/
```
~~~

I'm guessing this should be an ordered list. Also, used standard GFM literal blocks for consistency with the rest of the guide and to add bash syntax highlighting, and simplified the sentences a little.